### PR TITLE
#380 font family and other spellings

### PR DESCRIPTION
**Pull request by rsupnekar**

### CODE REVIEW

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java [hunk 1]
```diff
@@ -59,14 +59,14 @@
     private static ReentrantLock lock = new ReentrantLock();
 
     /**
-     * If the view hodling the Map is being resized then we may end up with GridPoints for many different width and height, so
+     * If the view holding the Map is being resized then we may end up with GridPoints for many different width and height, so
```
Comment (jrummel) [inline mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java]: widths and heights should be plural.

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java [hunk 2]
```diff
@@ -59,14 +59,14 @@
...
      * we must clean up if we have more than max allowed.
      */
     private static int MAX_GRID_POINTS = 10;
 
     /**
      * In the case where two corners of the view show sky, we may have to make an adjustment to the calculated polygon
-     * based on the ration of the side that is on the edge to the two adjacent perpandicular sides.
+     * based on the ration of the side that is on the edge to the two adjacent perpendicular sides.
```
Comment (jrummel) [inline mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java]: is ration the right word here?

File: sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java [hunk 3]
```diff
@@ -371,7 +371,7 @@ public double getAzimuth() {
     @Override
     public void setPathType(PathType pathType) {
         if (null == pathType) {
-            throw new IllegalArgumentException("Paramater ca not be null.");
+            throw new IllegalArgumentException("Parameter ca not be null.");
```
Comment (jrummel) [inline sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java]: can*

File: sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/FreehandDrawEditor.java [hunk 4]
```diff
@@ -36,7 +36,7 @@
  * 2) We listen for map and feature user interaction DRAG and DRAG_COMPLETE events.
  * 3) When the 2nd drag event occurs the draw start event is generated. And the line is drawn on the map
  * in the style indicated.
- * 4) When additional drag event are received we generate drag update eevents and the line feature is
+ * 4) When additional drag event are received we generate drag update events and the line feature is
```
Comment (jrummel) [inline sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/FreehandDrawEditor.java:39]: events

File: sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/AbstractMapGridLine.java [hunk 5]
```diff
@@ -177,7 +177,7 @@ public Date getLastUpdated() {
     @Override
     public List<IFeature> getGridFeatures() {
         // We need to return the feature in the return list so the map can loop thru it with out us
```
Comment (jrummel) [inline sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/AbstractMapGridLine.java]: without should be one word

**Review by jrummel — APPROVED** [review]

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java (hunk 2 above)
Comment (rsupnekar) [reply mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java]: I fixed it to ratio

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java (hunk 1 above)
Comment (rsupnekar) [reply mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/utils/BoundsGeneration.java]: done

File: sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java (hunk 3 above)
Comment (rsupnekar) [reply sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java]: fixed

File: sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/FreehandDrawEditor.java (hunk 4 above)
Comment (rsupnekar) [reply sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/FreehandDrawEditor.java:39]: this is ok

File: sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/AbstractMapGridLine.java (hunk 5 above)
Comment (rsupnekar) [reply sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/AbstractMapGridLine.java]: fixed

**Pull request merged**